### DISCUSSION TITLE
m2ts file not being picked up during scan?

### DISCUSSION THREAD

**Discussion by victorhooi** [stashapp/stash#4553] [Q&A] [answered]
**Describe the bug**
I have a .m2ts video file in the stash library directory, that does not appear to be picked up.

**To Reproduce**
1. Put a m2ts video file in the stash Library directory
2. Go to settings, then Tasks, then click on Scan

**Expected behavior**
The m2ts file should be picked up during the scan

**Stash Version: (from Settings -> About): v0.24.3-13-g1588d1cb

**Desktop (please complete the following information):**
 - OS: macOS 14.2.1
 - Browser [e.g. chrome, safari]: Chrome
 - Version: 123.0.6270.0 (Official Build) canary (arm64)

**Comment by HereIam80** [comment, marked as answer]
Did you add m2ts to the video extensions?

![image](https://github.com/stashapp/stash/assets/31324543/b590048b-4ef5-48cd-9e06-d7e6948f07a1)
